Rakefile

```diff
@@ -15,7 +15,6 @@ end
 Rake::TestTask.new do |t|
   t.libs << "."
   t.warning = true
-  t.verbose = true
```

Comment:
That change fixes the [CI build failure](https://travis-ci.org/ruby-json-schema/json-schema/builds/115218296) that I ran into the when I first submitted this PR. It looks like the verbose flag isn't supported by jruby or Rubinius.